docs/03-tutorials/00-serverless/svls-02-create-git-function.md

```diff
@@ -83,6 +83,8 @@ Follow these steps:
         type: # "basic" or "key"
         secretName: # "git-creds-basic" or "git-creds-key"
     ```
+   
+    >**NOTE:** It is recommended to use small repositories to source your git functions. Using large multi-repos can cause problems.
```

Comment:
What problems?
```suggestion
    >**NOTE:** It is recommended to use small repositories to source your git functions. Using large multi-repos can cause problems with function controller stability as it may run out of CPU and i/o resources.
```